flybywiresim/aircraft
# refactor(efb): flyPad OS 3.2 Throttle Configuration Refactor

Fixes #[issue_no]

## Summary of Changes

Refactor of the throttle page to make room for future expansion
Fixes bug with light theme and plane outline svg not being in the correct theme color
Adds svg elements for different aircraft in relevant EFB pages

## Screenshots (if necessary)



Before:
![FlightSimulator_xVOgxJfiz9](https://github.com/flybywiresim/aircraft/assets/15316958/f15d90c4-6649-4472-a67f-c1fc781d423b)

After - not on 320: 

![image](https://github.com/flybywiresim/aircraft/assets/15316958/9f4cdd48-eb76-4afe-857f-2dd581e1c805)


## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Go to EFB Settings > Sim Options > Calibrate Throttles
2. Check that TOGA, FLX, CLB, etc navigation bar works as expected when it is clicked
3. Check that independent throttle option and reverser on axis works and toggles the UI elements
4. Check that clicking the blue pencil edit (next to Current Value: ) changes the bottom UI element
5. Ensure that the front-end UI elements are all visible and no visual glitches are present
6. Change values for deadband +/-, press the edit button next to the Current Value to switch to detailed view, change inputs and ensure that no visual front-end glitches present themselves.
7. When deadband +/- is altered, ensure the recommended deadband value message is present. 
8. Change languages and ensure that this page continues to function well in your chosen language. 
9. Overlap the Idle and CLB detents (Set the same value for both) and ensure the error message is visible and not glitched.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
